crates/js-component-bindgen/src/intrinsics.rs

```diff
@@ -171,18 +171,10 @@ pub fn render_intrinsics(
                 const i64ToF64 = i => (i64ToF64I[0] = i, i64ToF64F[0]);
             "),
 
-            Intrinsic::InstantiateCore => match instantiation {
-                Some(InstantiationMode::Async) | None =>  {
-                    output.push_str("
-                        const instantiateCore = WebAssembly.instantiate;
-                    ")
-                }
-
-                Some(InstantiationMode::Sync) =>  {
-                    output.push_str("
-                        const instantiateCore = WebAssembly.Instance;
```

Comment:
Is this removal fixing a bug in the previous instantiation sync work? If so, should we also change https://github.com/bytecodealliance/jco/blob/main/crates/js-component-bindgen/src/transpile_bindgen.rs#L248?

Reply:
Do you mean the default value for `instantiateCore` should be removed?

Reply:
Oh no, I mean that `instantiateCore` is still being initialized to `WebAssembly.Instance` when it was removed here? So it's a consistency point for the usage.

Reply:
Specifically shouldn't we also change that definition to be `instantiateCore = WebAssembly.instantiate` instead of `instantiateCore = WebAssembly.Instance`?

Reply:
Okay thanks for clarifying - right, so it simply doesn't apply to this case.